styles.css

```diff
@@ -4,12 +4,45 @@ body {
 }
 
```

Comment:
- [ ] Kindly reset the margin and padding. You can add the following code on top of your stylesheet. This will get rid of the white space on the corners.

```
* {
margin: 0;
padding: 0;
}
```